crates/lnd-bridge/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust:1.77 as builder
```

Comment:
Why run and build it in a container?
This will slow down our CI builds significantly.

Reply:
This was a left over from when I used a builder image. This can be removed.

Reply:
Not sure if you understood me: building and running the mock in a container will slow down CI (provided we write tests for this).

Reply:
oh got ya. I thought you were referring to the `as builder` part. don't you think the CI will cache the build image as well? if not we could manually push an image and use this instead in the docker compose. It shouldn't change to often anyways.